web/src/unread_ops.js

```diff
@@ -36,6 +37,10 @@ const FOLLOWUP_BATCH_SIZE = 1000;
 // case after a server-initiated reload.
 let window_focused = document.hasFocus && document.hasFocus();
 
+// Since there's a database index on is:unread, it's a fast
+// search query and thus worth including here as an optimization.),
+const all_unread_messages_narrow = [{operator: "is", operand: "unread", negated: false}];
```

Comment:
kept only this condition as a seperate constant since it is also used to determine whether we need to clear unread.old_unreads_missing

Reply:
Yeah this is reasonable.